src/js/post-911-gib-status/routes.jsx

```diff
@@ -1,8 +1,13 @@
-import Post911GIBStatusApp from './containers/Post911GIBStatusApp';
+import React from 'react';
+import { Route } from 'react-router';
 
-const routes = {
-  path: '/',
-  component: Post911GIBStatusApp
-};
+import PrintPage from './containers/PrintPage';
+
+const routes = [
```

Comment:
Just browsed the above IndexRoute docs as well as [RouteConfiguration](https://github.com/ReactTraining/react-router/blob/v3/docs/guides/RouteConfiguration.md) and `IndexRoute` seems like the way to go.